off-chain/src/Network/Users.purs

```diff
@@ -50,10 +62,15 @@ sendActionToLeader
   :: forall a
    . UserNode a
   -> UserAction a
-  -> Effect $ Either IncludeActionError UUID
-sendActionToLeader = undefined
+  -> Aff $ Either IncludeActionError UUID
+sendActionToLeader userNode action =
+  (userHandlers userNode).submitToLeader action
 
-getActionStatus :: forall a. UserNode a -> UUID -> Effect StatusResponse
+-- ! misha: not sure, what it suppose to do
```

Comment:
The intention is to query the server about the action status.

If we find the action in one of the queues of the server, we send that back, otherwise we return a `not found`

Reply:
now that I think about that after the last change in interface, we should add a queue for `actions` that we weren't wrong about and whose the user hasn't asked about yet, and clear it every 10 minutes or something like that.

Reply:
> he intention is to query the server about the action status.

Got it.